README.md

```diff
@@ -1,54 +1,106 @@
-xblock-group-project-v2
-=======================
+# xblock-group-project-v2
+
 
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
-Configuration variables
------------------------
+## Features
 
-This block uses following configuration variables in the ``XBLOCK_SETTINGS``:
+* Structure - group project is organized around activities and stages. 
+    * Activities are the larger chunks, used to distinguish between logically bound parts of group project. Activities 
+      can contain one or more stages. Grading happens at activity level, i.e. one grade per activity.
+    * Stages are smaller building blocks, representing individual steps towards project completion.
+* Collaborative learning - students are organized into workgroups (aka cohorts) to work on a set of problems in a
+  collaborative way.
+* Team communication - group project interface allows sending emails to individual teammates or entire workgroup.
+* Private discussions - private (aka cohorted) discussions can be configured for the group prohect, to provide platform 
+  for discussing the assignments between teammates, and avoid spoiling the results/ideas to other workgroups or future
+  course students.
+* Resources - course authors can provide a set of resources - videos or documents - to help orient students, provide 
+  deliverable templates, or just share some relevant information about the assignment.
+* File submissions - outcomes are uploaded to the server, to facilitate sharing them between teammembers and for future
+  grading.
+* Peer feedback - workgroup members can provide anonymous feedback to each other.
```

Comment:
@e-kolpakov "outcomes are" ⟶ "work produced by students is"?

"teammembers" ⟶ "team members"